### PR TITLE
NO-ISSUE: add CPU vs response time panel, unify limits and resources

### PR DESCRIPTION
**Pull request by rccrdpccl**
Adding new panel CPU vs response time.
This will be useful once we can tweak limits and verify how the service is affected.

Current CPU and Memory usage charts:
current_cpu_usage
current_memory_usage

Proposed CPU and Memory usage charts:

proposed_cpu_usage
proposed_memory_usage

Additional chart CPU vs response time:
throttling_5m_interval

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml [hunk 1]
```diff
@@ -508,6 +508,135 @@ data:
...
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "gH4yqWBnz"
+              },
+              "editorMode": "code",
+              "expr": "avg(rate(container_cpu_usage_seconds_total {namespace=\"$namespace\",pod=~\"assisted-service-.*\"\n} [5m]))",
```
Comment (rccrdpccl) [inline dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml:577]: Ideally we should use variable rate time window, we'll follow up in a later PR to provide options as `$__rate_interval` is not suitable in this case (curve not smooth enough)

File: dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml [hunk 2]
```diff
@@ -607,7 +736,7 @@ data:
                 "uid": "${datasource}"
               },
               "editorMode": "code",
-              "expr": "sum(kube_pod_container_resource_limits{namespace=\"$namespace\", pod=~\"assisted-service-.*\", container=\"assisted-service\", resource=\"memory\"}) by (pod)",
```
[the remark this answers is not in the document]
Comment (rccrdpccl) [reply dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml:610]: This series is about memory limits: I do not think that having one line per pod would add any information, as we already show the current values by pod.
Normally limits don't change, but when they do, we will see the line gradually going up or down, depending on the change and on the stage of the rollout, and we can infer number of pods from other metrics.

File: dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml (hunk 1 above)
Comment (rccrdpccl) [reply dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml:577]: Example charts have selected 3 months view:

Chart with `$__rate_interval`:
throttling_rate_interval

Chart with `5m` interval hardcoded:
throttling_5m_interval

Ideally we should add a global variable to select interval, so we are able to change it when selecting larger/small timeframes, as `$__rate_interval` is not behaving as we'd like it to

Comment (nmagnezi) [reply dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml:577]: A Grafana bug?

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Pull request merged**